cmd/containerize/containerize.go

```
@@ -0,0 +1,67 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"teleport-exec/cgroups"
+	"teleport-exec/containerize"
+	"time"
+)
+
+func main() {
+	controller := containerize.NewController()
+	defer controller.Close()
+
+	if len(os.Args) < 2 {
+		fmt.Println("Usage: containerize.go command args...")
+		os.Exit(1)
+	}
+
+	// Prepare cgroups support
+	cgroups.Setup()
+	defer cgroups.TearDown()
```

you're ignoring the error return here